packages/optimistic-oracle/index.js

```diff
@@ -4,6 +4,7 @@ require("dotenv").config();
 const retry = require("async-retry");
 
 const { Logger, waitForLogger, delay, OptimisticOracleClient, GasEstimator } = require("@uma/financial-templates-lib");
+const { OptimisticOracleKeeper } = require("./src/keeper");
```

Comment:
do we want to call this a `Keeper`? would it not be more consistent to call it `OptimisticOracleProposer` or something to that effect so it's consistent with the other packages?

Reply:
Well its also a disputer, I think its fine to call it proposer though

Reply:
Renamed